### PR TITLE
iscsi plugin: print error messages not just exit status.

### PR DESCRIPTION
**Pull request by ghost**
Low risk since it is only changing the error messages of the iSCSI plugin.

### CODE REVIEW

**Comment by roberthbailey** [comment]
Please add a risk assessment to your PR description.

**Comment by swagiaal** [comment]
@roberthbailey I added a risk assessment to the PR description

**Comment by thockin** [comment]
LGTM  @quinton-hoole 

Low-risk, better errors are worthwhile.  Not required, so I could be overridden.

File: pkg/volume/iscsi/iscsi_util.go [hunk 1]
```diff
@@ -83,15 +83,15 @@ func (util *ISCSIUtil) AttachDisk(iscsi iscsiDisk) error {
...
-			glog.Errorf("iscsi: failed to sendtargets to portal %s error:%v", iscsi.portal, err)
+			glog.Errorf("iscsi: failed to sendtargets to portal %s error: %s", iscsi.portal, string(out))
 			return err
 		}
 		// login to iscsi target
-		_, err = iscsi.plugin.execCommand("iscsiadm", []string{"-m", "node", "-p", iscsi.portal, "-T", iscsi.iqn, "--login"})
+		out, err = iscsi.plugin.execCommand("iscsiadm", []string{"-m", "node", "-p", iscsi.portal, "-T", iscsi.iqn, "--login"})
 		if err != nil {
-			glog.Errorf("iscsi: failed to attach disk:Error: %v", err)
+			glog.Errorf("iscsi: failed to attach disk:Error: %v (%v)", string(out), err)
```
Comment (quinton-hoole) [inline pkg/volume/iscsi/iscsi_util.go]: Nit: why %v here and %s above.  Better to be consistent?

**Comment by quinton-hoole** [comment]
Very minor nit, but I think this is safe and valuable enough to merge.

**Comment by quinton-hoole** [comment]
i.e. LGTM, ok-to-merge as soon as the nit is addressed

**Comment by roberthbailey** [comment]
Also please make sure shippable is passing. 

@k8s-bot ok to test

File: pkg/volume/iscsi/iscsi_util.go (hunk 1 above)
Comment (swagiaal) [inline pkg/volume/iscsi/iscsi_util.go]: my bad.. I missed that.

**Comment by swagiaal** [comment]
@quinton-hoole I made the change you noted.

**Comment by quinton-hoole** [comment]
LGTM!  Thanks @swagiaal !

**Comment by a-robinson** [comment]
Shippable timed out. Re-kicking the build by closing and re-opening the PR.

**Comment by a-robinson** [comment]
Shippable is having serious problems today. This looks safe enough to merge without that visibility.

**Comment by swagiaal** [comment]
@a-robinson I see.. thanks for rerunning it

**Pull request merged**